### PR TITLE
[FLAVA]Use boolean pretrained flag to load ckpts

### PR DESCRIPTION
**Pull request by ankitade**
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #368
* #366
* __->__ #365
Using bool flag similar to omnivore since there is only one ckpt to load

Test plan
1. Pretraining debug
python -m flava.train config=flava/configs/pretraining/debug.yaml training.lightning.gpus=0 training.lightning.strategy=null
Epoch 0: : 50it [03:10,  3.81s/it, loss=16.3, v_num=3, train/losses/mmm_text_loss=10.50, train/losses/mmm_image_loss=9.150, train/losses/itm_loss=0.838, train/losses/global_contrastive_loss=2.090, train/losses/mlm_loss=10.40, train/losses/mim_loss=9.180]

2. Finetune debug
python -m flava.train config=flava/configs/pretraining/debug.yaml model.pretrained=True
Epoch 0: : 50it [00:28,  1.78it/s, loss=6.78, v_num=4, train/losses/mmm_text_loss=0.456, train/losses/mmm_image_loss=6.150, train/losses/itm_loss=0.378, train/losses/global_contrastive_loss=3.170,

3. coco zero shot
python -m flava.coco_zero_shot --data_root /datasets01/COCO/022719/val2017 --annotations /datasets01/COCO/022719/annotations/captions_val2017.json

Differential Revision: [D41142483](https://our.internmc.facebook.com/intern/diff/D41142483)

### CODE REVIEW

**Comment by ankitade** [comment]
@deankita has imported this pull request.  If you are a Meta employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D41142483).

File: torchmultimodal/models/flava/model.py [hunk 1]
```diff
@@ -505,15 +505,15 @@ def flava_model(
         image_projection=image_projection,
     )
 
-    if pretrained_model_key is not None:
-        flava.load_model(FLAVA_MODEL_MAPPING[pretrained_model_key])
+    if pretrained:
+        flava.load_model(FLAVA_MODEL_MAPPING["flava_full"])
```
Comment (ankitade) [inline torchmultimodal/models/flava/model.py:509]: i can move the flava_full string to constant in followup PR

**Comment by ankitade** [comment]
@deankita has imported this pull request.  If you are a Meta employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D41142483).

**Review by ebsmothers — APPROVED** [review]

**Comment by ankitade** [comment]
@deankita has imported this pull request.  If you are a Meta employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D41142483).
